packages/core/internal-schema.ts

```diff
@@ -47,6 +47,9 @@ export interface IInternalTextField extends IInternalField {
   missingValue?: string;
   minLength?: number;
   maxLength?: number;
+  hints: IFieldHints & {
+    layout: 'text' | 'textArea';
```

Comment:
Your change in the Schema docs says the values are "input" or "textArea".  I think the documentation is more accurate.  This should be fixed to match.

Reply:
Good catch I will log and add a bug to fix this.